### DISCUSSION TITLE
Dynamic import with custom server

### DISCUSSION THREAD

**Discussion by BrunoHenrique00** [vercel/next.js#41486] [Help]
Hey, I have a next app that uses the [custom server feature](https://nextjs.org/docs/advanced-features/custom-server) and now this app is actually so huge that we need to implement code-splitting and other performance fixes.

My question is, does `next/dynamic` work properly when using this custom server with next?
Do I need to make any specific configuration to make the `dynamic` work?

**Comment by dejurin** [comment]
Next.js supports lazy loading external libraries with import() and React components with next/dynamic.
